eslint.config.js

```diff
@@ -15,5 +17,26 @@ module.exports = [
         }
       ]
     }
+  },
+  // Configurations applied only to test files
+  {
+    files: [
+      'test/**/*.ts'
+    ],
+    languageOptions: {
+      ...testConfig.languageOptions
```

Comment:
This approach is not preferable since it requires knowledge of what objects `testConfig` contains. However, I cannot think of a more appropriate method.